code/network/multi_observer.cpp

```diff
@@ -59,7 +59,7 @@ void multi_obs_create_player(int player_num,char *name,net_addr *addr,player *pl
 	Net_players[player_num].last_heard_time = timer_get_fixed_seconds();
 	Net_players[player_num].reliable_socket = PSNET_INVALID_SOCKET;
 	Net_players[player_num].s_info.kick_timestamp = -1;
-	Net_players[player_num].s_info.voice_token_timestamp = -1;
+	Net_players[player_num].s_info.voice_token_timestamp = UI_TIMESTAMP::invalid();
 	Net_players[player_num].s_info.player_collision_timestamp = timestamp(0);
```

Comment:
Looks like you can convert this to use `TIMESTAMP`, and use `TIMESTAMP::immediate()`